README.md

```diff
@@ -301,6 +301,8 @@ We welcome PRs and suggestions! Don't hesitate to open a PR/issue or to reach ou
 Please have a look at our [contribution guide](CONTRIBUTING.md) and
 ["Help Wanted" issues](https://github.com/skyvern-ai/skyvern/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22) to get started!
 
+If you want to chat with the skyvern repository to get a high level overview of how it is structured, how to build off it, and how to resolve usage questions, check out [Code Sage](https://storia.ai?utm_source=github&utm_medium=referral&utm_campaign=skyvern-readme).
```

Comment:
https://storia.ai?utm_source=github&utm_medium=referral&utm_campaign=skyvern-readme

this link doesn't work for me

Reply:
Fixed in #770 . Sorry for that